contentcuration/contentcuration/templates/edit_page.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE HTML>
+{% extends "base.html" %}
+
+{% load staticfiles %}
+{% block content %}
+<div id="edit">
+<div id="button_bar">
+	<a id="move_button" href="#move" class="button">Move</a>
+	<a id="copy_button" href="#copy" class="button">Copy</a>
+	<a id="delete_button" href="#delete" class="button">Delete</a>
+	<a href="#clipboard_overview"class="button">Clipboard</a>
+</div>
+
+<div id="edit_container">
+	<canvas class="sidebar"></canvas>
+	<h2 class="title">Khan Academy</h2>
+	<div id="edit_list">
+		<ul>
+			<script type="text/javascript">
```

Comment:
javascript looks good, and i know it was written for prototyping purposes, but in the future, make sure you put any javascript that you write into a separate .js file. this will be important when it comes time to start working with backbone